### PR TITLE
win64

### PR DESCRIPTION
**Pull request by eggrobin**
Project and solution file changes, as well as:
https://github.com/mockingbirdnest/Principia/pull/929/files#diff-97c11ce73c03063f1dd5f89989bb1fff
https://github.com/mockingbirdnest/Principia/pull/929/files#diff-c8666bccb01bd25d11f7ae5e0b5f2d1f
https://github.com/mockingbirdnest/Principia/pull/929/files#diff-f525b5f8281758d601381cf1fba79587
https://github.com/mockingbirdnest/Principia/pull/929/files#diff-9bcbdc8113ca6210e64c0e03d945e93f
https://github.com/mockingbirdnest/Principia/pull/929/files#diff-cbca62329f704110e750a5cf650edb88
https://github.com/mockingbirdnest/Principia/pull/929/files#diff-19e8c6ec426b989b92a6926fc41118fb
Warnings [C4244](https://msdn.microsoft.com/en-us/library/th7a07tz.aspx) and [C4267](https://msdn.microsoft.com/en-us/library/6kck0s93.aspx) ignored for now, because they occur in Google headers.

### CODE REVIEW

File: ksp_plugin_adapter/ksp_plugin_adapter.cs [hunk 1]
```diff
@@ -152,6 +152,11 @@ private enum PluginSource {
     // platform problems in C++.
     System.IO.Directory.CreateDirectory("glog/Principia");
     try {
+      if (!Loader.LoadPrincipiaDll()) {
+        // TODO(egg): return a string from LoadPrincipiaDll and use it.
+        bad_installation_popup_ = "something happened";
```
Comment (pleroy) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: Please open an issue to track this.

File: ksp_plugin_adapter/loader.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  static extern IntPtr LoadLibrary(
+      [MarshalAs(UnmanagedType.LPStr)]string lpFileName);
+
+  const int RTLD_NOW = 2;
+  [DllImport("dl")]
+  static extern IntPtr dlopen(
+      [MarshalAs(UnmanagedType.LPTStr)] string filename,
+      int flags = RTLD_NOW);
+
+
```
Comment (pleroy) [inline ksp_plugin_adapter/loader.cs]: Remove extra empty line.

File: rebuild_all_solutions.ps1 [hunk 3]
```diff
@@ -1,13 +1,22 @@
...
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Debug .\Google\googlemock\msvc\2015\gmock.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Release .\Google\googlemock\msvc\2015\gmock.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Debug .\Google\protobuf\vsprojects\protobuf.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Release .\Google\protobuf\vsprojects\protobuf.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Debug .\Google\benchmark\msvc\google-benchmark.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Release .\Google\benchmark\msvc\google-benchmark.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Debug .\Principia\Principia.sln
-&$msbuild /t:"Clean;Build" /m /property:Configuration=Release .\Principia\Principia.sln
+$dependencies = @(".\Google\glog\google-glog.sln",
+               ".\Google\googletest\msvc\gtest.sln",
```
Comment (pleroy) [inline rebuild_all_solutions.ps1]: Bad indent.

**Comment by pleroy** [comment]
Please open an issue for the two warnings.

File: rebuild_all_solutions.ps1 (hunk 3 above)
Comment (eggrobin) [inline rebuild_all_solutions.ps1]: Done.

File: ksp_plugin_adapter/loader.cs (hunk 2 above)
Comment (eggrobin) [inline ksp_plugin_adapter/loader.cs]: Made them explicitly private and moved them to the bottom.

File: ksp_plugin_adapter/ksp_plugin_adapter.cs (hunk 1 above)
Comment (eggrobin) [inline ksp_plugin_adapter/ksp_plugin_adapter.cs]: #930.

**Comment by eggrobin** [comment]
Opened #931.

**Pull request merged**